### PR TITLE
Add perf package and missing admin ssh keys

### PR DESCRIPTION
**Pull request by brtknr**

### CODE REVIEW

File: ansible/group_vars/all/alaska [hunk 1]
```diff
@@ -116,5 +116,8 @@ cluster_admins:
...
     - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDtz8BtUJFQe1IMunfKA0pHSKOv8Xq5ehQ73aFAOQi/rb60X10MQvEdXlT5KOv0GfDB8VcJctVl0q7WIlzplrVfOMHn4L9Mgyko8uN/drT//6YtceGQdu+tAgogvYemLeieN5Ms7JJNUVBXE3h8wxYEY2O1h+ykeaB88IAG5iXzaP69WyAUdtun0ACu+8ixYESkzw18wVgVYcxKt8eehabHFsSP8eLPJOreTObIChcmR5Uq1D4sXl5nE7pmIwbAnd4bzVUdpjxYs/pjFPq4JujBF5HHBzFfyyaGVlq8GXFEtNEFMbYXLI9xQPXv/T+7NB+dLhdPGB/2w08sybyXkpqJ kbs@localhost.localdomain"
+    - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDpwtPMNbEDSGlk6eBsG2L8ytYolSaVA+rRkR8pHGG0L+Lpvc8IfrQvx7T1rqlpfz2uiACy629SHHK/HUmNHm3YuQM6M1hQrGNz38COFrg5ls0KOvcSZnAEMXmVR9wtKhVF2CZ/+Fz3HVBZXznFD+S8av1R06O5cgoB1sRkUqsjJZi3d1dKV87a4Ive5RpHDnueAlcLaZsEDyKh3ve2Ez8B++9A3I93V2zq/wlVC7c3MbRrJsfYcCo91hPZPDSQvpJ7C+NJx+B9G/ZHXzh0ko8fTRQebq8pQexleY1p4Ylmxyzlm+2G3VpnsJDNnG1SdZhqbD9qnYi8qfmTLrnuuQAB"
```
Comment (dougszumski) [inline ansible/group_vars/all/alaska]: Who's key do we have here?

File: config/openhpc.yml [hunk 2]
```diff
@@ -165,4 +165,5 @@ openhpc_packages:
   - readline-devel
   - python-devel
   - git-lfs
+  - perf
```
Comment (dougszumski) [inline config/openhpc.yml]: This is for Peter right?

File: ansible/group_vars/all/alaska (hunk 1 above)
Comment (brtknr) [reply ansible/group_vars/all/alaska]: Thats right, I've added a comment.

File: config/openhpc.yml (hunk 2 above)
Comment (brtknr) [reply config/openhpc.yml]: Yep!

Comment (dougszumski) [reply config/openhpc.yml]: And he was requesting `perl-devel` as well?

Comment (brtknr) [reply config/openhpc.yml]: Good point!

**Comment by brtknr** [comment]
Thanks!

File: ansible/group_vars/all/alaska [hunk 3]
```diff
@@ -116,5 +116,8 @@ cluster_admins:
...
     - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDtz8BtUJFQe1IMunfKA0pHSKOv8Xq5ehQ73aFAOQi/rb60X10MQvEdXlT5KOv0GfDB8VcJctVl0q7WIlzplrVfOMHn4L9Mgyko8uN/drT//6YtceGQdu+tAgogvYemLeieN5Ms7JJNUVBXE3h8wxYEY2O1h+ykeaB88IAG5iXzaP69WyAUdtun0ACu+8ixYESkzw18wVgVYcxKt8eehabHFsSP8eLPJOreTObIChcmR5Uq1D4sXl5nE7pmIwbAnd4bzVUdpjxYs/pjFPq4JujBF5HHBzFfyyaGVlq8GXFEtNEFMbYXLI9xQPXv/T+7NB+dLhdPGB/2w08sybyXkpqJ kbs@localhost.localdomain"
+    - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDpwtPMNbEDSGlk6eBsG2L8ytYolSaVA+rRkR8pHGG0L+Lpvc8IfrQvx7T1rqlpfz2uiACy629SHHK/HUmNHm3YuQM6M1hQrGNz38COFrg5ls0KOvcSZnAEMXmVR9wtKhVF2CZ/+Fz3HVBZXznFD+S8av1R06O5cgoB1sRkUqsjJZi3d1dKV87a4Ive5RpHDnueAlcLaZsEDyKh3ve2Ez8B++9A3I93V2zq/wlVC7c3MbRrJsfYcCo91hPZPDSQvpJ7C+NJx+B9G/ZHXzh0ko8fTRQebq8pQexleY1p4Ylmxyzlm+2G3VpnsJDNnG1SdZhqbD9qnYi8qfmTLrnuuQAB" # Peter Wortmann
```
Comment (markgoddard) [inline ansible/group_vars/all/alaska]: Why is Peter using the centos user rather than his own account?

Comment (brtknr) [reply ansible/group_vars/all/alaska]: He had his key added to the centos user previously as he wanted to try different configurations

**Pull request merged**